corehq/apps/reports/static/reports/js/reports.async.js

```diff
@@ -1,4 +1,4 @@
-hqDefine("reports/js/reports.async", function () {
+hqDefine("reports/js/reports.async", ['hqwebapp/js/alert_user'], function (alertUser) {
```

Comment:
nit whitespace to match existing conventions

```suggestion
hqDefine("reports/js/reports.async", [
    'hqwebapp/js/alert_user'
], function (alertUser) {
```